source/_docs/automation/trigger.markdown

```diff
@@ -47,6 +58,22 @@ automation:
 
 Similar to [script level variables](/integrations/script/#variables), `trigger_variables` will be available in trigger templates with the difference that only [limited templates](/docs/configuration/templating/#limited-templates) can  be used to pass a value to the trigger variable.
 
+The value of the `trigger_variables` are assigned at the time the automation is configured, and are often used during the configuration of the automation.
+
+The script level variables defined by the `variables` property are assigned each time the trigger is fired. The `trigger_variables` that were assigned at configuration time are also available to the templates used in your conditions and action. They do not get assigned a new value at the time the trigger is fired.  However, if you put the same variable name in `variables` and `trigger_variables` the value will be assigned again at trigger time using the template defined in the `variables` property.
```

Comment:
```suggestion
The script level variables defined by the `variables` property are assigned each time the trigger is fired. The `trigger_variables` that were assigned at configuration time are also available to the templates used in your conditions and action. They do not get assigned a new value at the time the trigger is fired. However, if you put the same variable name in `variables` and `trigger_variables` the value will be assigned again at trigger time using the template defined in the `variables` property.
```

Reply:
Additionally, what is "script level variables", I think that wording is confusing and unclear. Especially considering Home Assistant has scripts as well (which is a different thing).